trie/verkle.go

```diff
@@ -72,7 +72,9 @@ func (t *VerkleTrie) TryUpdateAccount(key []byte, acc *types.StateAccount) error
 	if err = t.TryUpdate(utils.GetTreeKeyNonce(key), nonce[:]); err != nil {
 		return fmt.Errorf("updateStateObject (%x) error: %v", key, err)
 	}
-	if err = t.TryUpdate(utils.GetTreeKeyBalance(key), acc.Balance.Bytes()); err != nil {
+	var balance [32]byte
+	copy(balance[32-len(acc.Balance.Bytes()):], acc.Balance.Bytes())
```

Comment:
`big.Int.Bytes()` returns the bytes in big-endian format so we want to reverse them, and then place them at the start of the 32-byte value.

Reply:
Now I am curious about the behavior of the tree when providing a value less than 32-bytes in length (or greater).